examples/sveltekit-pwa/pwa.js

```diff
@@ -0,0 +1,57 @@
+import { resolveConfig } from 'vite'
+import replace from '@rollup/plugin-replace'
+import { VitePWA } from 'vite-plugin-pwa'
+import { copyFileSync } from 'fs'
+import minimist from 'minimist'
+
+const args = minimist(process.argv.slice(2))
+
+process.env.CLAIMS = `${args['CLAIMS'] === 'true'}`
+process.env.RELOAD_SW = `${args['RELOAD_SW'] === 'true'}`
+process.env.SW = `${args['SW'] === 'true'}`
+
+const webmanifestDestinations = [
+  './.svelte-kit/output/client/',
```

Comment:
it'd probably be nice to use tabs in this file since the rest of the project does